File: PHP/Program/Aksesoris.php
<?php

require ('PetShop.php');

class Aksesoris extends PetShop
{
    // Atribut-atribut yang akan dipakai pada main.php
    private $jenis = "";
    private $bahan = "";
    private $warna = "";

    // CONSTRUCTOR dengan parameter
    public function __construct($jenis = "", $bahan = "", $warna = "")
    {
        parent::__construct();
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    // GETTER SETTER

    // Setter ID
    public function set_jenis($jenis)
    {
        $this->jenis = $jenis;
    }

    // Getter ID
    public function get_jenis()
    {
        return $this->jenis;
    }

    // Setter nama produk
    public function set_bahan($bahan)
    {
        $this->bahan = $bahan;
    }

    // Getter nama produk
    public function get_bahan()
    {
        return $this->bahan;
    }

    // Setter stok produk
    public function set_warna($warna)
    {
        $this->warna = $warna;
    }

    // Getter stok produk
    public function get_warna()
    {
        return $this->warna;
    }

    // DESTRUKTOR
    public function __destruct()
    {
        
    }
}

File: PYTHON/Program/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop):
      
    # ATRIBUT-ATRIBUT YANG MAU DIINISIALISASIKAN
    __jenis = ""
    __bahan = ""
    __warna = ""

     # CONSTRUCTOR DENGAN PARAMETER UNTUK ME-NEW KAN
    def __init__(self, jenis = "", bahan = "", warna = ""):
        self.__jenis = jenis
        self.__bahan = bahan
        self.__warna = warna

    # Getter
    def get_jenis(self): return self.__jenis
    def get_bahan(self): return self.__bahan
    def get_warna(self): return self.__warna

    # Setter
    def set_jenis(self, jenis): self.__jenis = jenis
    def set_bahan(self, bahan): self.__bahan = bahan
    def set_warna(self, warna): self.__warna = warna

    # DESTRUKTOR
    # def __del__(self):
    #     print("contoh jika pakai destructor")